### DISCUSSION TITLE
Sky130 commercial tutorial Place & Route fail

### DISCUSSION THREAD

**Discussion by johnw08** [ucb-bar/hammer#865] [Support (Q&A)]
Hello,

I'm following the Sky130 commercial tutorial and I'm using Genus and Innovus. The synthesis step works fine but place & route stuck at 2nd line of floorplan.tcl:

`place_inst system/tile_prci_domain/tile_reset_domain_tile/dcache/data/data_arrays_0/data_arrays_0_ext/mem_0_0 50 50 r90`

This line triggers the following error:

`**ERROR: (IMPTCM-162):	"system/tile_prci_domain/tile_reset_domain_tile/dcache/data/data_arrays_0/data_arrays_0_ext/mem_0_0" does not match any object in design for specified type "inst " object in command "place_inst".`

It seems like the object path was not set properly. Any help on where I could do wrong please? Thank you.

**Comment by harrisonliew** [comment]
This was fixed in ucb-bar/chipyard#1882.

**Reply by johnw08** [reply]
Thank you for the quick response. I cloned a new repository and checked out to this commit and re-run the tutorial. It did pass that line of code in floorplan.tcl file, but it failed on the next line.
```
@@file 10: place_inst system/tile_prci_domain/element_reset_domain_rockettile/dcache/data/data_arrays_0/data_arrays_0_ext/mem_0_0 50 50 r90

Instance system/tile_prci_domain/element_reset_domain_rockettile/dcache/data/data_arrays_0/data_arrays_0_ext/mem_0_0 is placed sideway.  Reset orientationto to R0. 
Flip instance system/tile_prci_domain/element_reset_domain_rockettile/dcache/data/data_arrays_0/data_arrays_0_ext/mem_0_0 to match row orient.

@@file 11: create_place_halo -insts system/tile_prci_domain/element_reset_domain_rockettile/dcache/data/data_arrays_0/data_arrays_0_ext/mem_0_0 -halo_deltas {2.0 2.0 2.0 2.0} -snap_to_site

**ERROR: (IMPFP-7695):	The Macro/BlackBox/CommitPtn/Block/IOPad/Cell 'system/tile_prci_domain/element_reset_domain_rockettile/dcache/data/data_arrays_0/data_arrays_0_ext/mem_0_0' was not found, so halo was not added.
```

**Reply by harrisonliew** [reply]
@nayiri-k can you help investigate?
